__tests__/rtFixtureLoaderHelper.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 Stanford University see LICENSE for license
```

Comment:
2019

Reply:
true, but I wonder if "2018, 2019" is actually more correct?  (not that it matters in this context)